engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -91,54 +92,61 @@ def execute_webhook(webhook_pk, alert_group_id, user_id):
         logger.warn(f"Webhook {webhook_pk} does not exist")
         return
 
-    data = _build_payload(webhook.trigger_type, alert_group_id, user_id)
+    try:
+        alert_group = AlertGroup.unarchived_objects.get(pk=alert_group_id)
+    except AlertGroup.DoesNotExist:
+        return
+
+    data = _build_payload(webhook.trigger_type, alert_group, user_id)
     status = {
-        "last_run_at": timezone.now(),
-        "input_data": data,
+        "timestamp": timezone.now(),
```

Comment:
what about setting `timezone.now` as the default on the model's `timestamp` attribute?